test/app-luatest/gh_5064_console_ignore_i_flag_without_tty_test.lua

```diff
@@ -0,0 +1,22 @@
+local t = require('luatest')
+local g = t.group()
+
+local result_str =  [[tarantool> 42
+---
+- 42
+...
+
+tarantool> ]]
+
+local TARANTOOL_PATH = arg[-1]
+
+g.test_console_ignores_i_flag_without_tty = function()
+    local cmd = [[printf '42\n' | ]] .. TARANTOOL_PATH .. [[ -i 2>/dev/null]]
```

Comment:
Nit: I think it would be good to hold `echo <...> | tarantool -i script.lua` behavior as well.

Reply:
The behavior in this case is unchanged by the patch, as it follows different if-else branch. It seems unfit to add such test in this commit.

Reply:
I would like to see it in this commit or a separate one, but I don't insist here.